src/Properties.cpp

```diff
@@ -17,7 +17,7 @@ int Properties::BAUD_RATE = 9600;
 
 // ********** MODULES ********** //
 boolean Properties::MODULE_WIFI = false;
-boolean Properties::MODULE_PI = false;
+boolean Properties::MODULE_PI = true;
```

Comment:
⚠️ This should be set to false when it is checked in. This would potentially be a breaking change if someone doesn't have a PI module installed on their device. It should be assumed that only the BASE Module is present. All others should be assumed to be absent.